plugin/advancedsubscription/src/AdvancedSubscriptionPlugin.php

```diff
@@ -0,0 +1,872 @@
+<?php
```

Comment:
Plugins should have name with underscore:

advanced_subscription  instead of advancedsubscription